user_sync/config.py

```diff
@@ -519,6 +543,10 @@ def get_rule_options(self):
         if options.get('directory_group_mapped'):
             options['directory_group_filter'] = set(six.iterkeys(self.directory_groups))
 
+        # set the adobe group filter from the mapping, if requested.
+        if options.get('adobe_group_mapped'):
```

Comment:
I would pass False as the second argument to get() to make it default to False if the key is not defined.